### ISSUE TITLE
Move editor group into next/previous group(Ctrl+Alt+ Left/Right arrow) commands are not working

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#53485]
Environment Details:
VSCode Version : 1.24.1

Additional Details:
MAS Violated: MAS2.1.1

Repro Steps:
1)Launch VS Code.
2)Open Keyboard Shortcuts.

Actual:
Move editor group into next/previous group(Ctrl+Alt+ Left/Right arrow) commands are not working. After using this command, screen starts rotating.

Expected:
Both the commands should work.

Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

MAS Reference
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

Attachment for Reference: 
![move editor group](https://user-images.githubusercontent.com/39204909/42216822-bbeef318-7ee0-11e8-8bd1-b497ba50bca7.JPG)

Does this issue occur when all extensions are disabled?: Yes

**Comment by bpasero** [comment]
Marking as "upstream" because this seems to be a keybinding the monitor takes for rotating. I do not think we can change anything about that.

**Comment by bpasero** [comment]
Closing, there is nothing we can do to detect this currently and keybindings can be configured.
